components/ipfs_ui/reducers/ipfs_reducer.ts

```diff
@@ -21,7 +21,7 @@ const openURlInNewTab = (target: string, api: string) => {
     return
   }
   const port = api.slice(api.lastIndexOf('/') + 1, api.length)
-  window.open(target.replace('{api-port}', port), '_blank')
+  window.open(target.replace('{api-port}', port), '_blank', 'noreferrer')
```

Comment:
cc @lidel @yrliou - i don't think this would break any ipfs functionality but please check

Reply:
cc @cypt4 to double check

Reply:
Checked, it is working